org-ql.el

```diff
@@ -27,6 +27,10 @@
 
 ;;;; Variables
 
+(if (fboundp 'org-timestamp--to-internal-time)
+    (defalias 'org-timestamp-to-time 'org-timestamp--to-internal-time)
```

Comment:
I think the `DEFINITION` argument should be sharp-quoted, i.e. `#'org-timestamp--to-internal-time`.